CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 2.12.0
+
+- Add `headers` config on `Client`
```

Comment:
Will there be an update to `README.md` describing how to use the `headers` config?

Reply:
I was thinking about that. Since it is more used by us internally I don't think we have to add a use case on README about this one. Opinions?